docs/admin/administration.md

```diff
@@ -622,14 +622,14 @@ that join the DAOS system. Once an engine has joined the DAOS system, it is
 identified by a unique system "rank". Multiple ranks can reside on the same
 host machine, accessible via the same network address.
 
-A DAOS system can be shutdown and restarted to perform maintenance and/or
-reboot hosts. Pool data and state will be maintained providing no changes are
+A DAOS system can be shut down and restarted to perform maintenance and/or
+reboot hosts. Pool data and state will be maintained, providing that no changes are
 made to the rank's metadata stored on persistent memory.
 
-Storage reformat can also be performed after system shutdown. Pools will be
-removed and storage wiped.
+Storage reformats can also be performed after a system shutdown. Pools
```

Comment:
keeping original - please revert
("storage reformat" is the dmg command, no plural)

Reply:
fixed